Point out our license (#4)
helpwave/.github: profile/README.md

Then simply write to us at: [contact@helpwave.de](mailto:contact@helpwave.de)

## Repositories

> > Unless otherwise defined in a repository of helpwave in the LICENSE file, the repository is under the Mozilla Public License 2.0.

Reviewer: ```suggestion
> Unless otherwise defined, all helpwave's repositories are under the Mozilla Public License 2.0.
```

Reviewer: The original sentence is a little bit weird structured.
